spyderlib/plugins/configdialog.py

```diff
@@ -761,21 +769,25 @@ def setup_page(self):
         margins_layout.addWidget(margin_spin)
         prompt_box = newcb(_("Prompt when exiting"), 'prompt_on_exit')
 
-        # Language chooser
-        choices = sorted([(val, key) for key, val in LANGUAGE_CODES.items()])
-        language_combo = self.create_combobox(_('Language'), choices,
-                                              'interface_language',
-                                              restart=True)
-
         # Decide if it's possible to activate or not singie instance mode
         if running_in_mac_app():
             self.set_option("single_instance", True)
             single_instance_box.setEnabled(False)
 
+        # Layout interface
+        comboboxes_layout = QHBoxLayout()
+        cbs_layout = QGridLayout()
+        cbs_layout.addWidget(style_combo.label, 0, 0)
+        cbs_layout.addWidget(style_combo.combobox, 0, 1)
+        cbs_layout.addWidget(icons_combo.label, 1, 0)
+        cbs_layout.addWidget(icons_combo.combobox, 1, 1)
+        cbs_layout.addWidget(language_combo.label, 2, 0)
+        cbs_layout.addWidget(language_combo.combobox, 2, 1)
+        comboboxes_layout.addLayout(cbs_layout)
+        comboboxes_layout.addStretch(1)

```

Comment:
What is the `1` for ? I think it's for the orientation, can't you use a `qt.something` constant instead ?